src/aliceVision/hdr/brackets.cpp

```diff
@@ -450,27 +456,133 @@ std::vector<std::vector<IndexT>> estimateGroups(const std::vector<LuminanceInfo>
         }
     }
 
-    //check coherency
-    bool coherency = true;
-    for (int idref = 1; idref < monotonics.size(); ++idref)
+    // In some cases (some Nikon cameras for instance) the medium exposure is at the first position of the ldr images group.
+    // Check that case and try to insert the resulting luminanceInfos seen as outliers at the mid index in the correponding groups.
+    if ((luminanceInfos.size() - monotonics.size() * monotonics[0].size() >= monotonics.size()) && // at least as many remaining outliers as groups
```

Comment:
monotonics[0].size() <--> bestBracketCount ?

Reply:
same think because above in the code, all groups with different size from bestBracketCount are removed.